test/e2e/network/util.go

```diff
@@ -49,3 +53,25 @@ func DescribeSvc(ns string) {
 		ns, "describe", "svc", fmt.Sprintf("--namespace=%v", ns))
 	framework.Logf(desc)
 }
+
+func VerifySCTPModuleLoadedOnNodes(f *framework.Framework, nodes *v1.NodeList) bool {
+	hostExec := utils.NewHostExec(f)
+	defer hostExec.Cleanup()
+	re := regexp.MustCompile(`^\s*sctp\s+`)
+	cmd := "lsmod | grep sctp"
+	for _, node := range nodes.Items {
+		framework.Logf("Executing cmd %q on node %v", cmd, node.Name)
+		result, err := hostExec.IssueCommandWithResult(cmd, &node)
+		if err != nil {
+			framework.Logf("sctp module is not loaded or error occured while executing command %s on node: %v", cmd, err)
+		}
+		for _, line := range strings.Split(result, "\n") {
+			if found := re.Find([]byte(line)); found != nil {
+				framework.Logf("the sctp module is loaded on node: %v", node.Name)
+				return true
```

Comment:
this will return once it finds an sctp module loaded and will not continue iterating over all the nodes.

Reply:
While there are some tests where it would be interesting to check specific nodes, the goal of the check is mostly "if *no* nodes have the SCTP module loaded before the test, then no nodes should have the SCTP module loaded after the test".

Reply:
yeah, but the current implementation is not doing that, is it?
it returns true on the first hit

Reply:
Now I updated the implementation. I collect the nodes with SCTP at the beginning, and then after the test, too. I compare the two lists and if there is a new node with SCTP after the test the test fails.
As it is not necessarily a 100% stable environment I had to do something with the cases when the interrogation of a node fails for any reason. The new logic works so, that if a node has SCTP module after the test, but the interrogation of that node failed before the test, then it is not considered as a new node with SCTP.
Please tell me if this logic is wrong.

Reply:
> yeah, but the current implementation is not doing that, is it?
> it returns true on the first hit

Right... because if any node has the module loaded, then that means that it is not the case that no node has the module loaded

Reply:
There is a very strong expectation that no node in the test environment will have the sctp module loaded. It's not worth adding a lot of complexity to deal with the case where some nodes to and some don't. Just require that if no node has the module loaded before the test, then no node has the module loaded after the test. (If at least one node has the module loaded before the test then it doesn't matter what the state after the test is.)

Reply:
OK, I reverted to the simpler solution.